elements/lisk-p2p/src/p2p.ts

```diff
@@ -43,6 +44,8 @@ import {
 	DUPLICATE_CONNECTION_REASON,
 	INCOMPATIBLE_PEER_CODE,
 	INCOMPATIBLE_PEER_UNKNOWN_REASON,
+	INVALID_CONNECTION_URL_CODE,
```

Comment:
since its about invalid address maybe address instead of url `INVALID_CONNECTION_ADDRESS_CODE`?